promises.cf.in

```diff
@@ -55,15 +55,15 @@ body common control
       # modules in which you MUST provide package modules used to generate
       # software inventory reports. You can also provide global default package module
       # instead of specifying it in all package promises.
-    debian.!disable_inventory_package_refresh::
+    (debian|windows).!disable_inventory_package_refresh::
           package_inventory => { $(package_module_knowledge.platform_default) };
 
       # We only define pacakge_invetory on redhat like systems that have a
       # python version that works with the package module.
     (redhat|centos|suse|sles|amazon_linux).cfe_yum_package_module_supported.!disable_inventory_package_refresh::
         package_inventory => { $(package_module_knowledge.platform_default) };
 
-    (debian|redhat|suse|sles|amazon_linux)::
+    (debian|redhat|suse|sles|amazon_linux|windows)::
```

Comment:
Same here, I think that for windows, you need the long class expression pegging it into the correct specific versions

Reply:
windows.(cfengine_3_12|cfengine_3_13|cfengine_3_14|cfengine_3_15).!cfengine_3_12_0::